.github/workflows/main_prover.yaml

```diff
@@ -69,13 +69,20 @@ jobs:
           cp configs/prover_client_config_example.toml configs/prover_client_config.toml
           cp configs/sequencer_config_example.toml configs/sequencer_config.toml
 
-          sed -i 's/sp1_deploy_verifier = false/sp1_deploy_verifier = true/' configs/sequencer_config.toml
-          sed -i 's/block_time_ms = 5000/block_time_ms = 12000/' configs/sequencer_config.toml
-          sed -i 's/listen_ip = "127.0.0.1"/listen_ip = "0.0.0.0"/' configs/sequencer_config.toml
-          sed -i 's/dev_mode = true/dev_mode = false/' configs/sequencer_config.toml
-
+          ETHREX_PROOF_COORDINATOR_DEV_MODE=false \
+          ETHREX_PROPOSER_BLOCK_TIME_MS=12000 \
+          ETHREX_DEPLOYER_PICO_DEPLOY_VERIFIER=true \
           make integration-test-gpu
 
       - name: Ensure admin permissions in _work
         if: always()
         run: sudo chown admin:admin -R /home/admin/actions-runner/_work/
+
+      - name: Report Status
```

Comment:
This should be removed, we no longer want to report this now that it runs on merge queues

Reply:
Solved in [03b779f](https://github.com/lambdaclass/ethrex/pull/2606/commits/03b779f87494b04ef28776f052c4ea70d5eb74b7).